### PR TITLE
pg: Fix error handling if bulkPortal execution fails

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

**Comment by mfussenegger** [comment]
@dobe please review

File: sql/src/test/java/io/crate/integrationtests/PostgresITest.java [hunk 1]
```diff
@@ -253,6 +253,74 @@ public void testExecuteBatch() throws Exception {
...
+            preparedStatement.addBatch();
+
+            preparedStatement.setString(1, "foobar");
+            preparedStatement.addBatch();
+
+            preparedStatement.setString(1, Integer.toString(3));
+            preparedStatement.addBatch();
+
+            try {
+                preparedStatement.executeBatch();
```
Comment (dobe) [inline sql/src/test/java/io/crate/integrationtests/PostgresITest.java]: pls assert the result

Comment (mfussenegger) [inline sql/src/test/java/io/crate/integrationtests/PostgresITest.java]: there is no result, this is throwing an exception

**Comment by dobe** [comment]
ack

**Pull request merged**